data/templates/router-advert/radvd.conf.j2

```diff
@@ -43,6 +43,13 @@ interface {{ iface }} {
     };
 {%             endfor %}
 {%         endif %}
+{%         if iface_config.source_address is vyos_defined %}
+    AdvRASrcAddress {
+{%             for source_address in iface_config.source_address %}
```

Comment:
loop not required as there will only be one source address as CLI does not have a `<multi/>` node for it.

Reply:
Ah, I missed the `<multi/>` then, since multiple addresses are possible according the radvd docs. I will add it.